packages/FetchableDataGrid/FetchableDataGrid.ts

```diff
@@ -250,6 +266,41 @@ export class FetchableDataGrid<TData, TDataFetcherParameters extends FetchableDa
 			</slot>
 		`
 	}
+
+	protected override get sidePanelTemplate() {
+		return html`
+			<mo-data-grid-side-panel
+				.dataGrid=${this as any}
+				tab=${ifDefined(this.sidePanelTab)}
+			>
+				<slot slot='settings' name='settings'>${this.settingsDefaultTemplate}</slot>
+				<slot slot='filter' name='filter'>
+					${this.filtersDefaultTemplate}
+
+					${[html.nothing, undefined].includes(this.filtersDefaultTemplate) || !this.hasFilters ? html.nothing : html`
+						<mo-button type='raised'
+							?disabled=${!this.hasParametersChanged}
+							@click=${() => this.restoreDefaultParameters()}
+						>
+							${t('Reset all filters')}
+						</mo-button>
+					`}
+				</slot>
+			</mo-data-grid-side-panel>
+		`
+	}
+
+	private get hasParametersChanged() {
+		return Object.entries(this.parameters ?? {})
+			.filter(([_, value]) => value !== undefined
+				&& value !== ''
+				&& (!(value instanceof Array) || value.length > 0))
+			.some(([key, value]) => JSON.stringify(this.initialParameters[key]) !== JSON.stringify(value))
+	}
+
+	private restoreDefaultParameters = () => {
```

Comment:
`restoreDefaultParameters` should be separate to ensure compatibility with future `ModdableDataGrid`.